packages/unmock-core/src/interfaces.ts

```diff
@@ -34,3 +34,27 @@ export interface IUnmockOptions {
   whitelist?: string[] | string;
   useInProduction?: boolean;
 }
+
+export interface ISerializedRequest {
+  body?: string;
+  headers?: any;
+  host: string;
+  method: string;
+  path: string;
+  protocol: "http" | "https";
+}
+
+export type IMockRequest = {
+  [P in keyof ISerializedRequest]?: ISerializedRequest[P] | RegExp;
+};
+
+export interface ISerializedResponse {
+  body?: string;
+  headers?: any;
```

Comment:
`headers?: { [key: string]: any }` might be a bit better typing wise

Reply:
Yes, I was lazily just copying the existing types.